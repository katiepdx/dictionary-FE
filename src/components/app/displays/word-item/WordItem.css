.label {
  font-weight: bold;
}

.wordItem {
  border: 2px solid black;
  font-size: 1.2rem;
  text-align: center;
}

.wordItem:hover {
  background-color: rgba(245, 222, 179, 0.637);
  transition: .6s;
}

a {
  text-decoration: none;
  color: black;
}
