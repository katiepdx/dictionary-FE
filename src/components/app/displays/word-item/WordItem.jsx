import React from 'react'
import PropTypes from 'prop-types'
import styles from './WordItem.css'

const WordItem = ({ word }) => {
  return (
    <div className={styles.wordItem}>
      <h4>Word: {word}</h4>
    </div>
  )
}

WordItem.propTypes = {
  word: PropTypes.string.isRequired
}

export default WordItem
