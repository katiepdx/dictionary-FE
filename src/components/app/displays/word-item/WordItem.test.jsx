import React from 'react'
import { render } from '@testing-library/react'
import WordItem from './WordItem'

describe('<WordItem/>', () => {
  it('should render a word item to the page', () => {
    const { asFragment } = render(<WordItem word="WordItemTest" />)

    expect(asFragment()).toMatchSnapshot()
  })
})
