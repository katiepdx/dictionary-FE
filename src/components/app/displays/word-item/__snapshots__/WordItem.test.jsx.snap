// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WordItem/> should render a word item to the page 1`] = `
<DocumentFragment>
  <div
    class="wordItem"
  >
    <h4>
      Word: Word
    </h4>
  </div>
</DocumentFragment>
`;
